tests/bluetooth/tester/src/bttester.c

```diff
@@ -260,6 +261,56 @@ static uint8_t *recv_cb(uint8_t *buf, size_t *off)
 	return new_buf->data;
 }
 
+#if defined(CONFIG_UART_PIPE)
+/* Uart Pipe */
+static void uart_init(uint8_t *data)
+{
+	uart_pipe_register(data, BTP_MTU, recv_cb);
+}
+
+static void uart_send(uint8_t *data, size_t len)
+{
+	uart_pipe_send(data, len);
+}
+
+#else
+#define UART_DEV	"UART_0"
+static uint8_t *recv_buf;
+static size_t recv_off;
+static const struct device *dev;
+K_TIMER_DEFINE(timer, timer_expiry_cb, NULL);
```

Comment:
timer_expiry_cb not declared yet.

Reply:
Sorry to disturb you. I'll confirm the compilation this time.